backups/restore.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Function to restore OpenMRS database
+function restore_openmrs_database() {
+  if [[ -d "$restore_folder/openmrs" ]]; then
+    echo "Restoring OpenMRS database"
+    docker exec -i "$openmrs_container" mysql -u $openmrsdb_username --password=$openmrsdb_password openmrs < "$openmrs_backup_file"
```

Comment:
Hope you have tested for restoring the stored procedures

Reply:
Yes @SooryaKumaranC-tw we tested, able to restore the stored procedure.

Reply:
Okay. Thanks